### ISSUE TITLE
git: Setting credential manager in post install of Git

### ISSUE THREAD

**Issue by dwettstein** [ScoopInstaller/Main#1617]
https://github.com/ScoopInstaller/Main/blob/4bbc182dabb2809bb1d5be4643608d231f59b99a/bucket/git.json#L16

What is the reason for setting the credential manager in post install of Git? IMHO, this is unnecessary and it could possibly overwrite a users existing (different) setting.

**Comment by rasa** [comment]
Agreed. Imo, this should only be set, if it is not currently set (globally).

**Comment by FriendlyNeighborhoodShane** [comment]
Does not need to be (read: should not be) set in any case _at all_. Git for Windows comes with the default set to a GUI program to help the user select one in case they don't already have it set in their user config, and this breaks that too.

I've explained the problem in more detail in #2190 (which is a dupe I created without searching first).

What is the reason this issue has been open for so long without any contact?

Are the maintainers looking for additional help? I'm sorry, I haven't really been keeping up.
